src/proposals/Proposals.jl

```diff
@@ -248,5 +249,145 @@ function (prop::RStagger)(rng::AbstractRNG,
     return u, v, logl, ncall
 end
       
+"""
+    Proposals.Slice(;slices=5, scale=1)
+
+Propose a new live point by a series of random slices away from an existing live point.
+This is a standard _Gibbs-like_ implementation where a single multivariate slice is a combination of `slices` univariate slices through each axis.
 
+## Parameters
+- `slices` is the minimum number of slices
+- `scale` is the proposal distribution scale, which will update _between_ proposals.
+"""
+@with_kw mutable struct Slice <: AbstractProposal
+    slices = 5
+    scale = 1.0
+    
+    @assert slices ≥ 1 "Number of slices must be greater than or equal to 1"
+    @assert scale ≥ 0 "Proposal scale must be non-negative"
+end
+
+function (prop::Slice)(rng::AbstractRNG,
+                       point::AbstractVector,
+                       logl_star,
+                       bounds::AbstractBoundingSpace,
+                       loglike,
+                       prior_transform;
+                       kwargs...)
+    # setup
+    n = length(point)
+    nc = nexpand = ncontract = 0
+    local idxs, r, u, u_prop, v_prop, logl_prop, logl_l, logl_r
+    
+    # modifying axes and computing lengths
+    axes = prop.scale .* Bounds.axes(bounds)
+    axlens = Bounds.axlens(bounds)    
```

Comment:
Sorry, I just changed some things, you'll have to rebase and then change this
```suggestion
    axes, axlens = Bounds.decompose(bounds)
    axes = prop.scale .* axes
```

Reply:
Ok